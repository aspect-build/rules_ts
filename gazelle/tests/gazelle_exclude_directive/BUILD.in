# gazelle:exclude main.ts
# gazelle:exclude subdir/ignored.ts
# gazelle:exclude **/*.ig.ts