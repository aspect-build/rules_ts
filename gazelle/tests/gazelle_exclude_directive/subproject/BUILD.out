# gazelle:ignore sub-ignored.ts